src/bluetooth.cpp

```diff
@@ -1746,12 +1746,12 @@ void bluetooth::connectedAndDiscovered() {
 
                 connect(ftmsAccessory, &smartspin2k::debug, this, &bluetooth::debug);
 
-                connect(this->device(), SIGNAL(resistanceChanged(int8_t)), ftmsAccessory,
-                        SLOT(changeResistance(int8_t)));
-                connect(this->device(), SIGNAL(resistanceRead(int8_t)), ftmsAccessory,
-                        SLOT(resistanceReadFromTheBike(int8_t)));
-                connect(ftmsAccessory, SIGNAL(resistanceRead(int8_t)), this->device(),
-                        SLOT(resistanceFromFTMSAccessory(int8_t)));
+                connect(this->device(), SIGNAL(resistanceChanged(int16_t)), ftmsAccessory,
```

Comment:
here we need to use Q_DECLARE_METATYPE and qRegisterMetaType https://stackoverflow.com/questions/19397567/passing-structure-to-signal-in-qt

Reply:
Okay, but resistance_t, which I have missed the replacement to here, is just a #define to one of the standard int types, not a struct, which is that that documentation refers to. Aren't these SIGNAL and SLOT macros obsolete? Are the connected slots not working for this code?

Reply:
Perhaps you were thinking ahead, perhaps thinking of actually having a resistance struct, which encapsulates a resistance level, and a resistance type (e.g. peloton, percentage, device), with converter functions like toPeloton(), toPercentage(), toDevice().

Reply:
> Okay, but resistance_t, which I have missed the replacement to here, is just a #define to one of the standard int types, not a struct, which is that that documentation refers to. 

but we can simply transform it to a typedef and it should work.
i can test it myself but not sooner than the next week


> Aren't these SIGNAL and SLOT macros obsolete? 

nope. at least i'm not aware of this 

> Are the connected slots not working for this code?

this code is working for sure, it's just a matter of cleaning the code :)

Reply:
> Perhaps you were thinking ahead, perhaps thinking of actually having a resistance struct, which encapsulates a resistance level, and a resistance type (e.g. peloton, percentage, device), with converter functions like toPeloton(), toPercentage(), toDevice().

nope, i mean it makes sense but i was thinking simply about the signal slot thing ;)